src/admin/includes/formDefinitions/callbacks.php

```diff
@@ -0,0 +1,24 @@
+<?php
+function processInsert(){
+  $localvars = localvars::getInstance();
+  $formInfo  = $_POST['MYSQL'];
+
+  $roomID = isset($formInfo['roomID']) ? $formInfo['roomID'] : NULL;
+
+  $dupCheck  = reservationPermissions::duplicatePermissionsCheck($formInfo['resourceID'], $formInfo['resourceType'], $roomID, $formInfo['email']);
```

Comment:
You don't need to waste the memory saving this variable. it can be done in an if statement.